gaj23/face-it
# Refactor for Bug Fixes and Media Queries

#### Purpose: 
> Check all that apply.

### What does this PR do? 
> Clearly and concisely describe changes made. 

Added media queries for desktop screens and refactored Details component to allow refresh and error handling.

### Problems Encountered & Resulting Solutions  
> State any problems encountered 

We had help from three different upper mod students and struggled with the refactoring piece for a while.  We finally figured out that the data was not being called at the right time to render what we needed it to on the display. 

> Describe your code changes in detail for reviewers.

The Details component includes hooks with useEffect and useState.  CSS files include media queries.

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

Details.js and all css files should have the appropriate changes.

### How should this be tested?
> What steps need to be taken to confirm functionality? 

Open the project in the browser, run `npm start` to start the server in conjunction with our backend server (run `node server.js` to access the data) and click through the pages to land on the final page of a single item.  Hit refresh to test the refresh functionality.  You can also enter a wrong id for the URL to render the Error component. 

### Steps To Take
> Any questions and possible steps moving forward.

It needs fine-tuning but it works and it's pretty user-friendly, I think! 

#### Checklist:
